tests/quarkus/src/test/java/io/hawt/tests/quarkus/suites/TestRunner.java

```diff
@@ -1,26 +1,14 @@
 package io.hawt.tests.quarkus.suites;
 
-import io.quarkiverse.cucumber.CucumberOptions;
-import io.quarkiverse.cucumber.CucumberQuarkusTest;
+import io.cucumber.junit.CucumberOptions;
 import io.quarkus.test.junit.QuarkusTest;
 
-import org.junit.jupiter.api.Test;
-
-import com.codeborne.selenide.Selenide;
-
 @QuarkusTest
 @CucumberOptions(
     features = "src/test/resources/features",
     glue = {"io.hawt.tests.quarkus", "io.hawt.tests.utils.stepdefinitions"},
     tags = "@quarkusAllTest",
-    plugin = {"json"})
-public class TestRunner extends CucumberQuarkusTest{
-    public static void main(String[] args) {
-        runMain(TestRunner.class, args);
-    }
-
-    @Test
-    public void testHelloEndpoint() {
-        Selenide.open("https://www.google.com");
-    }
+    plugin = {"json"},
+objectFactory = CucumberQuarkusTest.CdiObjectFactory.class)
```

Comment:
bridge Cucumber and Quarkus CDI - the CucumberOptions from quarkus-cucumber should do it... on the right quarkus version :sweat_smile: